src/components/Lend/LoanSearch/LoanSearchSavedSearch.vue

```diff
@@ -97,6 +101,10 @@ export default {
 			'view-new-filter-saved-search',
 			''
 		);
+
+		if (this.$route?.query?.saved_search ?? false) {
+			this.openModal();
```

Comment:
Which use case needs this modal opening when the page loads?

Reply:
Looks like the doneUrl setup in the loginUrl below is the case that would trigger this.

@roger-in-kiva I think you should add the `userId` to this if statement.

Reply:
We need to open the modal when coming back from login page triggered by openModal method.

The openModal method is considering the userId, if null is redirecting to login page. So if we trigger this as logged out via url query, the user will be redirected to login page automatically. Do you like to avoid this? @mcstover

Reply:
That works!